src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1468,6 +1468,27 @@ public static int Sign(float value)
             throw new ArithmeticException(SR.Arithmetic_NaN);
         }
 
+        // Helper functions for CopySign
+        // >= 0: returns 1
+        // < 0: returns -1
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static int SignZeroToOne(int value)
```

Comment:
This name isn't "clear" IMO and needs something that makes its meaning clear.

Reply:
SignOrOneIfZero?